src/build/contracts/Members.json

```diff
@@ -0,0 +1,19739 @@
+{
```

Comment:
`src/build` should be in .gitignore

Reply:
should be or should not be? On my current version there is a src/build in the .gitignore file 

![Screenshot from 2019-10-05 17-23-37](https://user-images.githubusercontent.com/44396280/66257087-1df13080-e795-11e9-8626-e88e8ef85098.png)

Reply:
The files in `src/build` should be ignored by git, however they are contained in the pull request. Looks like the gitignore is not working properly in this case.

Reply:
just a / missing :)